docs/spec-find.md

```diff
@@ -83,44 +83,41 @@ input StringInput {
   lt: String
   ge: String
   gt: String
-  in: [String]
+  in: [String!]
```

Comment:
`in` should be `[Type!]` instead of `[Type]` to prevent users from including nulls in the list